lib/admin_plugins/cleanentriesdb.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var moment = require('moment');
+
+var cleanentriesdb = {
+  name: 'cleanentriesdb'
+  , label: 'Clean Mongo entries (glucose entries) database'
+  , pluginType: 'admin'
+};
+
+function init() {
+  return cleanentriesdb;
+}
+
+module.exports = init;
+
+cleanentriesdb.actions = [
+    {
+        name: 'Delete all documents from entries collection older than 180 days'
+      , description: 'This task removes all documents from entries collection that are older than 180 days. Useful when uploader battery status is not properly updated.'
+      , buttonLabel: 'Delete old documents'
+      , confirmText: 'Delete old documents from entries collection?'
+      , preventClose: true
+    }
+  ];
+
+cleanentriesdb.actions[0].init = function init(client, callback) {
+  var translate = client.translate;
+  var $status = $('#admin_' + cleanentriesdb.name + '_0_status');
+
+  $status.hide();
+
+  var numDays =  '<br/>'
+      + '<label for="admin_entries_days">'
+      + translate('Number of Days to Keep:')
+      + '  <input id="admin_entries_days" value="180" size="3" min="1"/>'
+      + '</label>';
+
+  $('#admin_' + cleanentriesdb.name + '_0_html').html(numDays);
+
+  if (callback) { callback(); }
+};
+
+cleanentriesdb.actions[0].code =  function deleteOldRecords(client, callback) {
+  var translate = client.translate;
+  var $status = $('#admin_' + cleanentriesdb.name + '_0_status');
+  var numDays = Number($('#admin_entries_days').val());
+
+  if (isNaN(numDays) || (numDays < 1)) {
+    alert(translate('%1 is not a valid number', { params: [$('#admin_entries_days').val()] }));
```

Comment:
string is not yet in languages.js

Reply:
please check other strings as well

Reply:
Interesting... that's not a new string, as it's in the cleanstatusdb.js.  I'll check them all to ensure they are in the language.js.

Thanks!